src/getWeekYear/index.ts

```diff
@@ -2,6 +2,11 @@ import startOfWeek from '../startOfWeek/index'
 import toDate from '../toDate/index'
 import toInteger from '../_lib/toInteger/index'
 import requiredArgs from '../_lib/requiredArgs/index'
+import {
+  WeekStartOptions,
+  LocaleOptions,
+  FirstWeekContainsDate,
+} from 'src/types'
```

Comment:
Absolute paths won't work without tuning `tsconfig.json`.

```suggestion
  FirstWeekContainsDateOptions,
} from '../types'
```